Assets/scripts/BattleManager.cs

```diff
@@ -63,11 +64,19 @@ private class PlayerBattleData {
 	private PlayerBattleData attacker;
 	private PlayerBattleData defender;
 	private int battleProgressInMeasures; // Number of measures into the battle
-	private MeshRenderer targetLine; // Renderer for note targets. Currently a temporary black line.
+	private MeshRenderer targetLine; // Renderer for note targets. Currently a temporary black line.
+	private GameObject divider; // Divider between player's notes
 
 	public int battleMeasures = 1;
 	public Camera parentCam;
-	public GameObject notePrefab;
+	public GameObject notePrefab;
+
+	//Constants
+	private float CENTER_MARGIN = Screen.width * 2 / 27f;
+	private float UNIT_MARGIN = Screen.width * 3.5f / 27f;
+	private float FRET_RANGE = Screen.width / 3f; //TODO: Change based on number of players
+	private float SPAWN_HEIGHT = Screen.height;

```

Comment:
I'm not totally convinced screen width and height should be factored into cont values since the screen can be resized, but I can't seem to break it, so it's probably ok.